schemas/capability.json

```diff
@@ -518,6 +524,29 @@
       },
       "additionalProperties": false
     },
+    {
+      "properties": {
+        "dmxRange": {},
+        "type": { "enum": ["EffectDuration"] },
+        "comment": {},
+        "menuClick": {},
+        "switchChannels": {},
+        "helpWanted": {},
+        "duration": { "$ref": "definitions.json#/entities/time" },
+        "durationStart": { "$ref": "definitions.json#/entities/time" },
+        "durationEnd": { "$ref": "definitions.json#/entities/time" }
+      },
+      "oneOf": [
+        { "required": ["duration"] },
+        { "required": ["durationStart"] }
+      ],
+      "not": { "required": ["duration", "durationStart"] },
```

Comment:
Shouldn't be needed, since the `oneOf` above does not fire when both `required`s match.